incident.go

```diff
@@ -89,6 +89,7 @@ type Incident struct {
 	Body                 IncidentBody         `json:"body,omitempty"`
 	IsMergeable          bool                 `json:"is_mergeable,omitempty"`
 	ConferenceBridge     *ConferenceBridge    `json:"conference_bridge,omitempty"`
+	IncidentResponders   []IncidentResponders `json:"incidents_responders,omitempty"`
```

Comment:
Could you implement a test that ensures this field is being passed along in the JSON within the request body?

Reply:
@theckman Thanks for comment. I added the test. It is not documented but you can check response body using `Try it`. This field is must to test if additional responders were added successfully (if you want to stick to go-pagerduty and do not want to create your own http requests)

Reply:
Thank you for that context. The policy set by PagerDuty on this library is that only fields that are explicitly listed in the documentation are merged in.

I'll label this PR as being focused on an undocumented feature, and try to engage with PagerDuty to see if they'd be willing/able to document the field(s). I'm not an employee, so I'll probably end up filing a support ticket to make the request to have the documentation updated.

Reply:
@theckman Thank you!